scripts/update-editors-choice-apps-inputlist

```diff
@@ -1,27 +1,27 @@
 org.mozilla.firefox
 org.gnome.Builder
 com.github.johnfactotum.Foliate
-org.libreoffice.LibreOffice
 org.kde.kdenlive
 com.slack.Slack
 org.inkscape.Inkscape
 org.blender.Blender
 org.darktable.Darktable
-org.videolan.VLC
 org.gabmus.hydrapaper
-org.filezillaproject.Filezilla
 org.kde.krita
-org.gimp.GIMP
-com.spotify.Client
-org.shotcut.Shotcut
 org.audacityteam.Audacity
-com.getpostman.Postman
 org.eclipse.Java
 io.dbeaver.DBeaverCommunity
-org.gnome.FeedReader
 com.github.bilelmoussaoui.Authenticator
 org.openshot.OpenShot
 org.kde.okular
 im.riot.Riot
 net.scribus.Scribus
-org.gnome.PasswordSafe
\ No newline at end of file
+org.gnome.PasswordSafe
+org.gnome.design.Contrast
+com.uploadedlobster.peek
+com.vscodium.codium
```

Comment:
I think this is a terrible addition, it poorly works with a sandbox.

Reply:
Which of the three s a terrible addition? Feel free to remove it :)

Reply:
Just vscode, done.